docs/pages/careers/developer-advocate.md

```diff
@@ -50,28 +50,29 @@ Our products empower React developers to build awesome applications faster – w
 
 Depending on the day, you'll:
 
+- **Product**.
+  - Fix simple small annoyances in the product.
+  - Follow GitHub issues to understand where developers face frustration and develop strategies to overcome these. This could be suggesting or implementing documentation updates, or proposing or contributing code changes that solve the core issue.
+  - Give feedback to product management, to influence the product roadmaps based on developers' needs.
+- **Docs**.
+  - Rework the pages of the documentation that are confusing, base on feedback.
 - **Content**.
+  - Create inspiring sample applications, documentation, and tutorials.
+  - Write code where required to support how-to content, blog posts, and presentations.
   - Produce high-quality technical "how-to" content (blogs, webinars, demos, talks) addressing common user needs, latest technology advances, and emerging best practices. Videos might go to [our YouTube channel](https://www.youtube.com/@MUI_hq).
   - Distribute content and leverage pieces to broaden awareness of the MUI brand, via existing connections.
-  - Rework the pages of the documentation that are confusing, base on feedback.
 - **Community**.
   - Built our developer-focused community.
   - Deliver presentations at meetups, conferences, and other ecosystem events.
   - Create & produce events (podcasts, roundtables, hackathons).
   - Manage the sponsorships of events, newsletters, etc.
   - Help other team members grow at engaging with the community. For instance by encouraging and providing critical feedback on the blog posts they create, or by keeping track of the health of community contributions.
-- **Relationship-building**.
-  - Create relationships with fellow ecosystem influencers and open-source leaders.
-  - Nurture ongoing connections with user group members to build personal relationships and deeply understand their needs, usage, journeys, and barriers to adoption.
-- **Technical**.
-  - Write code where required to support how-to content, blog posts, and presentations.
-  - Create inspiring sample applications, documentation, and tutorials.
-  - Give feedback to product management, to influence the product roadmaps based on developers' needs.
-  - Follow GitHub issues to understand where developers face frustration and develop strategies to overcome these. This could be suggesting or implementing documentation updates, or proposing or contributing code changes that solve the core issue.
+  - Create relationships with fellow ecosystem influencers and open-source leaders. Nurture ongoing connections with user group members to build personal relationships and deeply understand their needs, usage, journeys, and barriers to adoption.
 
 For the right candidate:
 
-- Working with the Leadership to construct and execute on a hiring plan to grow the Developer Experience team.
+- Lead the Developer Experience team.
```

Comment:
We could potentially discuss this better on Notion, but just flagging that we should be clearer as far as what "Lead" means here ⎯ no need to answer me, but we should make sure candidates know whether this means managing people.

Reply:
I have added it under "For the right candidate" to broaden the scope of people who can apply. What you said sounds good